.github/workflows/pull-request.yml

```diff
@@ -1,134 +1,93 @@
 name: Pull Request
 
-defaults:
-  run:
-    working-directory: ./rust
-
 on:
-  pull_request:
   push:
-    branches: [master]
+    branches: '**'
```

Comment:
Can we flip this to just auto-build on `master` for now. I'd like to not burn through our Actions mins budget until we have the turbo caching in place. Otherwise, that 10 mins of build time is going to start adding up quick. We can ensure the turbo caching PR turns this back on for all branches.